src/pickup.cpp

```diff
@@ -803,9 +803,16 @@ void Pickup::pick_up( const tripoint &p, int min, from_where get_items_from )
                 draw_item_info( w_item_info, dummy );
             }
             draw_custom_border( w_item_info, 0 );
+
+            // print info window title: < item name >
             mvwprintw( w_item_info, point( 2, 0 ), "< " );
-            trim_and_print( w_item_info, point( 4, 0 ), itemsW - 8, c_white, "%s >",
+            int item_len = utf8_width( remove_color_tags( selected_item.display_name() ) );
+            if( item_len > itemsW - 8 ) {
+                item_len = itemsW - 8;
+            }
+            trim_and_print( w_item_info, point( 4, 0 ), item_len, selected_item.color_in_inventory(),
                             selected_item.display_name() );
+            mvwprintw( w_item_info, point( item_len + 4, 0 ), " >" );
```

Comment:
```suggestion
            wprintw( w_item_info, " >" );
```
This should print the text directly after the previous string (the display name).